<launch>


<!--	<node name="tf_scan1" pkg="tf" type="static_transform_publisher" args="0.9505 -0.6285 0.68 -0.785398163 0.0 0.0 /base_link /laser2 100" respawn="false" output="screen"/> -->

	<node name="tf_scan1" pkg="tf" type="static_transform_publisher" args="0.9505 -0.6285 0.68 -0.855398163 0.0 0.0 /base_link /laser2 100" respawn="false" output="screen"/>

	<node name="tf_scan2" pkg="tf" type="static_transform_publisher" args="-0.9505 0.6285 0.68 2.35619449 0.0 0.0 /base_link /laser1 100" respawn="false" output="screen"/>

	<node name="tf_imu" pkg="tf" type="static_transform_publisher" args="0.0 0.6285 0.68 0.0 0.0 0.0 /base_footprint /imu 100" respawn="false" output="screen"/>



  <node name="hokuyo1" pkg="hokuyo_node" type="hokuyo_node" respawn="false" output="screen">
    <remap from="scan" to="scan1"/>
    <param name="frame_id" type="string" value="/laser1"/>
    <!-- Starts up faster, but timestamps will be inaccurate. --> 
    <param name="calibrate_time" type="bool" value="true"/> 
    
    <!-- Set the port to connect to here -->
    <param name="port" type="string" value="/dev/ttyACM1"/> 
  
    <param name="intensity" type="bool" value="false"/>
    <param name="min_ang" value="-2.35619449"/>
    <param name="max_ang" value="2.35619449"/>
  </node>

  <node name="hokuyo2" pkg="hokuyo_node" type="hokuyo_node" respawn="false" output="screen">
      <remap from="scan" to="scan2"/>
      <param name="frame_id" type="string" value="/laser2"/>
    <!-- Starts up faster, but timestamps will be inaccurate. --> 
    <param name="calibrate_time" type="bool" value="true"/> 
    
    <!-- Set the port to connect to here -->
    <param name="port" type="string" value="/dev/ttyACM0"/> 
  
    <param name="intensity" type="bool" value="false"/>
    <param name="min_ang" value="-2.35619449"/>
    <param name="max_ang" value="2.35619449"/>
  </node>


	<node pkg="omnix" type="omnix_node" name="omnix_node" output="screen">
	<remap from="base_link" to="base_footprint"/>	      
	</node>

  	<node pkg="joy" type="joy_node" name="ps3_joy" >
    	<param name="dev" type="string" value="/dev/input/js0" />
    	<param name="deadzone" value="0.12" />
  	</node>

	<node pkg="omnix" type="omnix_ps3joy" name="omnix_ps3joy" output="screen">
	</node>



</launch>